jdaviz/configs/imviz/tests/test_catalogs.py

```diff
@@ -112,6 +112,8 @@ def test_plugin_image_with_result(self, imviz_helper, tmp_path):
         assert catalogs_plugin.results_available
         assert catalogs_plugin.number_of_results > 500
         prev_results = catalogs_plugin.number_of_results
+        table_last_ra = float(catalogs_plugin.table.items[-1]['Right Ascension (degrees)'])
+        assert_allclose(table_last_ra, imviz_helper.app._catalog_source_table[-1]['ra'], atol=0.0001)  # noqa
```

Comment:
Hmm... I copied just these 2 lines to the same test file on `main` and it does not fail without your patch. I am not sure if this added test demonstrates that the bug has been fixed.

Reply:
That's why it was still in draft. I just pushed a test that actually fails on main.

Reply:
Oh, was it in draft? 😅